content/en/docs/contribute/review/reviewing-prs.md

```diff
@@ -124,6 +124,12 @@ When reviewing, use the following as a starting point.
 ### Language and grammar
 
 - Are there any obvious errors in language or grammar? Is there a better way to phrase something?
+  - You should focus on the language and grammar of the parts of the page that is being changed.
+    Unless the author is clearly aiming to update the whole of an existing page, it's OK for them not
+    to fix everything that could be improved in it.
+  - If you are reviewing a pull request and you spot grammar, spelling or other style errors that
+    are not related to the PR's purpose, consider logging an issue about fixing those other issues.
+    (Check that there isn't an existing issue about this first).
```

Comment:
```suggestion
    (Check that there isn't an existing issue about this first.)
```
I think the period belongs inside the parens, since it is a full sentence and not a part of an existing sentence.

Reply:
I'd say get rid of the parentheses entirely, they should be checking for duplicates to avoid loading the triage folks